src/darwin/Framework/CHIP/MTRBaseDevice.mm

```diff
@@ -475,8 +475,21 @@ - (void)subscribeWithQueue:(dispatch_queue_t)queue
                                   }];
 }
 
+static NSDictionary<NSString *, id> * _MakeDataValueDictionary(NSString * type, id _Nullable value, NSNumber * _Nullable dataVersion)
+{
+    if (value && dataVersion) {
+        return @ { MTRTypeKey : type, MTRValueKey : value, MTRDataVersionKey : dataVersion };
```

Comment:
Does clang-format format it like this? Having a space between the `@ {` looks rather confusing...

Reply:
I agree with you, but unfortunately restyled made it this way.